app-modules/portal/src/Filament/Pages/ManagePortalSettings.php

```diff
@@ -143,6 +143,7 @@ public function form(Form $form): Form
                         SpatieMediaLibraryFileUpload::make('favicon')
                             ->collection('portal_favicon')
                             ->visibility('private')
+                            ->visible(Gate::check(Feature::KnowledgeManagement->getGateName()))
```

Comment:
Sorry, I should have made this one more precise. We want it to be controlled by the KB toggle above like the rest of the inputs in this form.

```suggestion
                            ->visible(fn (Get $get) => $get('knowledge_management_portal_enabled'))
```

Reply:
It's Okay. No Problem.